corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -114,7 +114,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
             var menuData = menusUtils.getMenuData(menuResponse);
             menuData["triggerEmptyCaseList"] = true;
             menuData["sidebarEnabled"] = true;
-            menuData["title"] = menuResponse.resultsTitle;
```

Comment:
The main change in the [original PR](https://github.com/dimagi/commcare-hq/pull/33261/files) was to add `results-title` field to suite file and then use it from `menuResponse`. Since we are no longer using it can we remove `results-title` from suite file now ?

Reply:
Yes we can. I was thinking of handling the reversions in a separate PR just make this simpler and get it out for this deploy, but it makes sense to just do it now for the HQ and formplayer side too. [eb7f65f](https://github.com/dimagi/commcare-hq/pull/33359/commits/eb7f65f504691fca7b29aa779f09249ece3cd4fa)

Just double checking, any built suites will still have the `results-title`, but what you found previously was that formplayer will just skip any unknown nodes so it'll be fine.